presto-hive/src/test/java/com/facebook/presto/hive/metastore/thrift/MockHiveMetastoreClient.java

```diff
@@ -306,4 +306,11 @@ public void close()
     {
         // No-op
     }
+
+    @Override
+    public List<String> setUGI(String userName, List<String> groupNames)
+    {
+        groupNames.add(userName);
```

Comment:
why do you modify input `paratemeter`. Maybe instead of this please return a copied list.

Reply:
thx, I will fix it